<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IN-7714GK3" failures="1" tests="4" name="Surefire test" time="7.219" errors="0" timestamp="2024-08-12T15:07:05 IST">
  <testcase classname="api.test.DataDrivenTests" name="testPostUser" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.NullPointerException">
      <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
Caused by: java.lang.NullPointerException
at api.utilities.XLUtility.getRowCount(XLUtility.java:38)
at api.utilities.DataProviders.getAllData(DataProviders.java:17)
... 30 more
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testPostUser -->
  <testcase classname="api.test.DataDrivenTests" name="testDeleteUserByName" time="2.89"/>
  <testcase classname="api.test.DataDrivenTests" name="testDeleteUserByName" time="1.394"/>
  <testcase classname="api.test.DataDrivenTests" name="testDeleteUserByName" time="1.432"/>
</testsuite> <!-- Surefire test -->
